sos/report/plugins/xdp.py

```diff
@@ -0,0 +1,20 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
```

Comment:
Looks like Travis didn't run, but this seems to be over 79 characters, so we'll need to wrap it for PEP8.

Reply:
Perhaps it's time we added an `__all__` member to `sos.report.plugins`? It's not everyone's stylistic cup-of-tea but I use it heavily in other projects that are designed to use `*` imports and shared namespaces.